cfme/ssui/myservice.py

```diff
@@ -0,0 +1,73 @@
+from widgetastic.widget import Text
+from widgetastic_manageiq import SSUIlist
+from widgetastic_patternfly import Input, Button
+from navmazing import NavigateToAttribute, NavigateToSibling
+from utils.update import Updateable
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ssui import navigator, SSUINavigateStep, navigate_to
+import cfme.fixtures.pytest_selenium as sel
+
+from . import DashboardView
+
+
+class MyServiceForm(DashboardView):
+    title = Text('#explorer_title_text')
+
+    service = SSUIlist(list_name='serviceList')
+    name = Input(name='name')
+    description = Input(name="description")
+
+    save_button = Button('Save')
+    cancel_button = Button('Cancel')
+
+
+class MyServicesView(MyServiceForm):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer and \
+            self.my_services.is_opened
+
+
+class DetailsMyServiceView(DashboardView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer and self.my_services.is_opened and \
+            self.title.text == '"{}"'.format(self.context['object'].name)
+
+
+class MyService(Updateable, Navigatable):
```

Comment:
This is what we were talking about above. There shouldn't be multiple MyService objects. Just one that has both ssui and ui components via Sentaku.